spec/models/membership_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Membership, type: :model do
+  describe 'basic validations' do
+    it { is_expected.to belong_to(:user) }
+    it { is_expected.to belong_to(:group) }
+    it { is_expected.to validate_presence_of(:user) }
+    it { is_expected.to validate_presence_of(:group) }
+  end
+
+  describe 'group draw and user draw must match' do
+    it do
+      user = FactoryGirl.create(:student)
+      group = FactoryGirl.create(:open_group)
+      membership = FactoryGirl.build(:membership, user: user, group: group)
+      expect(membership.valid?).to be_falsey
```

Comment:
Is this because the user has no draw? Would it be better to specify a different draw? I feel like we don't necessarily need to persist here, just stub out the `draw` method for both the `group` and `user` objects.